hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -880,9 +880,11 @@
               { value: "4",
                 name: "MACRO_WRITE_BLANK_ERROR",
                 desc: '''
-                A blank write check has failed during an OTP write operation. This effectively
-                aborts the write operation such that no harm is done to the OTP. The corresponding
-                controller automatically recovers from this error when issuing a new command.
+                This error is returned if a programming operation attempted to clear a bit that has previously been programmed to 1.
+                The corresponding controller automatically recovers from this error when issuing a new command.
+
+                Note however that the affected OTP word may be left in an inconsistent state if this error occurs.
+                This can cause unrecoverable errors upon the next reboot (depending on the OTP partition that the word belongs to).
```

Comment:
This might cause unrecoverable errors in OTP_CTRL's background checks as well right, is that worth mentioning that?

Reply:
wait, actually does that affect background check? Will background check check ECC for every word in buffered partition? Or does it only check digest values?

Reply:
Hm, technically this is true if the partition is locked.
But probably you run into errors much earlier than that, since you still need to lock the partition, which triggers a readout for the digest.
So probably I should rephrase this a bit and just say that it causes problems when this word is accessed again (either as part of a read op or as part of a check).